.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -1,6 +1,6 @@
 ---
-name: 'Wordlist request'
-about: 'Suggest a wordlist to be added'
+name: 'Wordlist/feature request'
```

Comment:
If you capitalize the word `feature`, this should be ready for merging.
`/feature` -> `/Feature`

Reply:
I’m not sure about this, as @g0tmi1k reverted some of my changes before that is related to the capitalisation issues.

https://github.com/danielmiessler/SecLists/commit/b4edd29c3c85005c2448dfca1981767c002803fc

Reply:
Yeah, I think I made a mistake. Sorry @molangning and thanks @ItsIgnacioPortal